man/en/sos-clean.1

```diff
@@ -0,0 +1,69 @@
+.TH SOS CLEAN 1 "Thu May 21 2020"
+.SH NAME
+sos clean - Obfuscate sensitive data from one or more sosreports
+.SH SYNOPSIS
+.B sos clean TARGET [options]
+    [\-\-domains]
+    [\-\-map]
+    [\-\-jobs]
+    [\-\-no-update]
+
+.SH DESCRIPTION
+\fBsos clean\fR or \fBsos mask\fR is an sos subcommand used to obfuscate sensitive information from
+previously generated sosreports that is not covered by the standard plugin-based post
+processing executed during report generation, for example IP addresses.
+.LP
+Data obfuscated via this utility is done so consistently, meaning for example an IP address of
+192.168.1.1 in an unprocessed sosreport that gets obfuscated to, for example, 100.0.0.1, will be
+changed to 100.0.0.1 in all occurences found in the report.
```

Comment:
Maybe add a note that, by default, the same obfuscation mapping is persisted over subsequent runs of sos clean?

Reply:
Ack.